test/vmulcaddc-microkernel-tester.h

```diff
@@ -255,3 +247,173 @@ class VMulCAddCMicrokernelTester {
   uint8_t qmax_{255};
   size_t iterations_{15};
 };
+
+#define XNN_TEST_VMULCADDC_ROW_DIV(ukernel, arch_flags, row_tile, channels_tile, datatype, params_type, init_params)   \
+  TEST(ukernel, ROW_div)                                                                                               \
+  {                                                                                                                    \
+    for (size_t rows = row_tile * 2; rows <= row_tile * 4; rows += row_tile) {                                         \
```

Comment:
`TEST_REQUIRES_ARCH_FLAGS(arch_flags)`? Does this pass tests locally for you?

Reply:
Thanks for pointing out the missing TEST_REQUIRES_ARCH_FLAGS(arch_flags). The issue with the illegal instruction for AVX-512 has now been resolved and now the tests are passing.